ops-bedrock/demo-docker-compose.yml

```diff
@@ -0,0 +1,263 @@
+version: '1'
+
+# This Docker compose file is intended to be used with the multi-rollup Espresso Sequencer demo
+# It assumes that a sequencer network and a geth node is running on the external network "espresso-sequencer".
+
+networks:
+  espresso-sequencer:
+    name: espresso-sequencer
+    external: true
+
+volumes:
+  op_l2_data:
+  op_log:
+  op_blockscout_db:
+  op_blockscout_logs:
+  op_blockscout_redis:
+
+services:
```

Comment:
At some point we should clean up all the various docker-compose files by having one definition of each service and using `extends` to bring each service into each compose setup. But this is not urgent.